### PR TITLE
using a smaller icon with thinner box shadow so the new focus styles fit

### PR DESCRIPTION
**Pull request by dlockhart**
Screen Shot 2019-05-29 at 5 13 07 PM

### CODE REVIEW

File: d2l-input-search.js [hunk 1]
```diff
@@ -35,44 +35,43 @@ $_documentContainer.innerHTML = `<dom-module id="d2l-input-search">
...
 
 			d2l-button-icon {
-				--d2l-button-icon-min-height: 1.7rem;
-				--d2l-button-icon-min-width: 1.7rem;
+				--d2l-button-icon-min-height: 1.5rem;
+				--d2l-button-icon-min-width: 1.5rem;
 				--d2l-button-icon-border-radius: 4px;
+				--d2l-button-icon-focus-box-shadow: 0 0 0 1px #ffffff, 0 0 0 3px #006fbf;
 				position: absolute;
-				height: 1.7rem;
```
Comment (svanherk) [inline d2l-input-search.js]: Was this just overkill?  I've seen this needed for IE11 in the past, but `d2l-button-icon` should probably be handling any issues with that

Comment (dlockhart) [reply d2l-input-search.js]: Seems to have been overkill, yes. Adjusting the `min-height` and width variables should (and does) affect the actual height and width.

**Comment by dlockhart** [comment]
I'm seeing some problems with the vertical positioning of the button that seems to be a pixel rounding thing. Will investigate further.

**Comment by dlockhart** [comment]
OK -- line-height fixes to `d2l-button-icon` plus a tweak to how we vertically centre the search button has this looking good in all browsers.

**Pull request merged**